tests/testsuite/build.rs

```diff
@@ -1226,8 +1226,7 @@ versions that meet the requirements `>= 1.0.1, <= 2.0.0` are: 2.0.0, 1.0.1
 
 all possible versions conflict with previously selected packages.
 
-  previously selected package `bad v2.0.1`
```

Comment:
Hm is this a bug that this clause no longer mentions `bad`? Reading the new error message it seems like it's not as clear as it was before

Reply:
definitely a regression, I'll see what we can do...

Reply:
Fixed.